modules/rpc/yarpc.go

```diff
@@ -49,6 +50,7 @@ type YARPCModule struct {
 	log       ulog.Log
 	stateMu   sync.RWMutex
 	isRunning bool
+	di        dig.Graph
```

Comment:
I'm sure you had your reasons, but I don't know if it's the right thing to attach di object to the YARPC module. Can we think of alternative approaches? Is this purely to make testing easier?

Wouldn't it make testing the handlers difficult, as you'd have to either:

a. Rely on the assumption that we fall back onto the default graph
b. Pass around the reference to the module